jenkins/ompi/ompi_test.sh

```diff
@@ -377,549 +331,566 @@ function oshmem_runner()
-            export ucx_root=$WORKSPACE/ucx_local
-            git clone https://github.com/openucx/ucx -b $jenkins_test_ucx_branch $ucx_root
-            (cd $ucx_root;\
-            ./autogen.sh;\
-            ./contrib/configure-release --prefix=$ucx_root/install;\
-            make -j9 install; )
+
+        if [ "${ci_test_use_ucx_branch}" = "yes" ]
+        then
+            export ucx_root="$WORKSPACE/ucx_local"
+            git clone https://github.com/openucx/ucx -b ${ci_test_ucx_branch} "${ucx_root}"
+            (cd "${ucx_root}";\
+                ./autogen.sh;\
+                ./contrib/configure-release --prefix="${ucx_root}/install";\
+                make -j"$(nproc)" install; )
            export UCX_DIR=$ucx_root/install
 
            # We need to override LD_LIBRARY_PATH because.
            # `module load hpcx-gcc-stack` will pull the legacy
            # UCX files that will interfere with our custom-built
            # UCX during configuration and the runtime I guess
-           export LD_LIBRARY_PATH=${HPCX_UCX_DIR}/lib:$LD_LIBRARY_PATH
+           export LD_LIBRARY_PATH="${HPCX_UCX_DIR}/lib:${LD_LIBRARY_PATH}"
         fi
-        export ucx_dir=$HPCX_UCX_DIR
+
+        export ucx_dir=${HPCX_UCX_DIR}
     fi
 
-    rm -rf $ompi_home_list
+    rm -rf "${ompi_home_list}"
 
     # build ompi
-    $autogen_script
-    echo ./configure $configure_args --prefix=$OMPI_HOME1 | bash -xeE
-    make $make_opt install
-    jenkins_build_passed=1
+    ${autogen_script}
+    echo "./configure ${configure_args} --prefix=${OMPI_HOME1}" | bash -xeE
+    make "${make_opt}" install
+    ci_build_passed=1
 
     # make check
-    if [ "$jenkins_test_check" = "yes" ]; then
-        make $make_opt check || exit 12
+    if [ "${ci_test_check}" = "yes" ]
+    then
+        make "${make_opt}" check || exit 12
     fi
 fi
 
-if [ -n "$jenkins_build_passed" ]; then
+if [ -n "${ci_build_passed}" ]
+then
     # check coverity
-    if [ "$jenkins_test_cov" = "yes" ]; then
-        vpath_dir=$WORKSPACE
-        cov_proj="all oshmem ompi/mca/pml/yalla ompi/mca/coll/hcoll"
-        if [ "$jenkins_test_ucx" = "yes" ]; then
-            cov_proj="$cov_proj ompi/mca/pml/ucx"
+    if [ "${ci_test_cov}" = "yes" ]
+    then
+        vpath_dir="$WORKSPACE"
+        cov_proj="all oshmem ompi/mca/coll/hcoll"
+
+        if [ "${ci_test_ucx}" = "yes" ]
+        then
+            cov_proj="${cov_proj} ompi/mca/pml/ucx"
         fi
-        cov_stat=$vpath_dir/cov_stat.txt
-        cov_stat_tap=$vpath_dir/cov_stat.tap
-        cov_build_dir=$vpath_dir/cov_build
-        cov_url_webroot=${JOB_URL}/${BUILD_ID}/Coverity_Report
-
-        rm -f $cov_stat $cov_stat_tap
-
-        if [ -d "$vpath_dir" ]; then
-            mkdir -p $cov_build_dir
-            pushd $vpath_dir
-            for dir in $cov_proj; do
-                if [ "$dir" = "all" ]; then
+
+        cov_stat="${vpath_dir}/cov_stat.txt"
+        cov_stat_tap="${vpath_dir}/cov_stat.tap"
```

Comment:
we used tap plugin in jenkins. in azure i think tap manipulation can be dropped.

Reply:
Fixed.